model/plugin_valid.go

```diff
@@ -15,7 +15,8 @@ const (
 )
 
 // ValidId constrains the set of valid plugin identifiers:
-//  ^[a-zA-Z0-9-_\.]+
+//
```

Comment:
Why there is an empty line needed here?

Reply:
This empty line is required for the new comment formatting of Go 1.19. The new `gofmt` now automatically formats code snippets with an empty line before and after them.